update.go

```diff
@@ -378,10 +388,18 @@ func initUpdate() error {
 		}
 
 		rc := http.NewResponseController(w)
-		start := time.Now()
-		killSupervisedServicesAndUmountPerm(signalDelay)
-		fmt.Fprintf(w, "All services killed in %s\n", time.Since(start))
-		rc.Flush()
+
+		if r.FormValue("async") == "true" {
+			log.Println("Asynchronous power off requested")
+			go func() {
+				killSupervisedServicesAndUmountPerm(signalDelay)
```

Comment:
This branch is still missing the “All services killed” message. In fact, maybe we should take this opportunity to refactor these identical lines into a shared function, so that the two places cannot go out of sync?

Reply:
Oh it looks like I need to rebase. Will do it tomorrow